### ISSUE TITLE
Not working on arch linux

### ISSUE THREAD

**Issue by MoSchaub** [buzz/volctl#37]
fails to start with the error importlib.metadata.PackageNotFoundError: volctl

**Comment by buzz** [comment]
If you'd provide more specifics, I might be able to help you.

**Comment by Narrat** [comment]
Kinda sounds like a missing rebuild for python 3.9

**Comment by MoSchaub** [comment]
How can I get the missing rebuild?

**Comment by Narrat** [comment]
You probably need to do it as this is only available from the AUR. And those are built from source.
Rebuild it with makepkg or the AUR helper of your choice

**Comment by dotfoo** [comment]
> If you'd provide more specifics, I might be able to help you.

I'll provide it: 

    Traceback (most recent call last):
      File "/usr/bin/volctl", line 33, in <module>
        sys.exit(load_entry_point('volctl==0.8.0', 'gui_scripts', 'volctl')())
      File "/usr/bin/volctl", line 22, in importlib_load_entry_point
        for entry_point in distribution(dist_name).entry_points
      File "/usr/lib/python3.9/importlib/metadata.py", line 524, in distribution
        return Distribution.from_name(distribution_name)
      File "/usr/lib/python3.9/importlib/metadata.py", line 187, in from_name
        raise PackageNotFoundError(name)
    importlib.metadata.PackageNotFoundError: volctl

I did a fresh pull and rebuilt, with the same result. I also happen to have older version (0.6.3-1) around, and it throws the same error. 

I did a huge update yesterday, (many Python packages where upgraded), so can't really do some simple downgrades to see where the error is coming from. 

Hope this helps, and will be glad to see it working soon again.

**Comment by buzz** [comment]
@dotfoo

The `volctl` Python package is not found on your system. Without knowing your Linux distro and installation method, it's only a shot in the dark. If you're on Arch, just use whatever [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers) suits you or clone the repo and use `makepkg` manually.

On other distros you should probably clone the repo and `sudo ./setup.py install` manually, use a virtualenv or use another method of installation which is appropriate for your distro.

**Comment by dotfoo** [comment]
Oh, sure, I am coming from an Arch system, and had built the package from AUR. Sorry that I wasn't clear about this yet. 

Unfortunately I'm not close to this machine and can't the packet again. I did realise that I actually didn't rebuild the package, and rather only reinstalled it. 

I'll let you know once I've rebuilt it again.

**Comment by dotfoo** [comment]
Allright, I did a proper rebuild now it is working properly again. Sorry for the hassle. 

Would you mind to bump the revision number over in AUR? I don't use any AUR helper myself, but I guess this might be happening on other machines as well.

**Comment by buzz** [comment]
I'm not sure what is the right thing to do to force a rebuild in case of a Python version upgrade. Bumping the revision number would still leave a time window in which the package would break for users unless AUR package authors would closely monitor the Python package version and bump in the exact right moment.

I found [this](https://www.reddit.com/r/archlinux/comments/5mer2f/psa_rebuild_all_aur_packages_providing_python/) which is the exact same case (for Python 3.5 to 3.6 switch). Personally I just rebuilt my AUR Python 3.8 packages by looking into what was left inside `/usr/lib/python3.8/site-packages`. Pretty much what is also suggested in the aforementioned thread.

If there's some recommended, foolproof way of automating rebuilds, I'll be happy to add it to the package.
